src/apolo_app_types/helm/apps/ingress.py

```diff
@@ -115,8 +114,8 @@ async def get_http_ingress_values(
         ingress_vals["annotations"][
             "traefik.ingress.kubernetes.io/router.middlewares"
         ] = (
-            f"{namespace}-{forward_auth_name}@kubernetescrd,"
-            f"{namespace}-strip-headers@kubernetescrd"
+            "platform-control-plane-ingress-auth@kubernetescrd,"
```

Comment:
the name in dev != name in prod clusters.

we need a conditional logic here depending whether we are in dev or prod